working/macros/feature-specification.md

```diff
@@ -871,23 +891,32 @@ Each argument in the metadata annotation for the macro application is converted
 to a form that the corresponding constructor on the macro class expects, which
 it specifies through parameter types:
 
-*   If the parameter type is `bool`, `double`, `int`, `Null`, `num`, or `String`
-    (or the nullable forms of any of those), then the argument expression must
-    be a Boolean, number, null, or string literal. Number literals may be
-    negated. String literals may not contain any interpolation, but may be
-    adjacent strings.
+*   If the parameter type is `bool`, `double`, `int`, `Null`, `num`, `String`,
+    `List`, or `Map`, (or the nullable forms of any of those), then the argument
```

Comment:
How about Set?

Reply:
Hmm, good question. I am not sure what the use case really would be given that you can't combine collections at all (at least for now). I am inclined to just leave it off to simplify things? But it wouldn't cause any issue to allow it afaik.

Reply:
We use them in mockito at least so that in the IDE, users see when they've accidentally duplicated something.

Reply:
I went ahead and added it